sys-apps/osinfo-db-tools/osinfo-db-tools-1.11.0.ebuild

```diff
@@ -0,0 +1,52 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_{10..12} )
+
+inherit meson python-any-r1
+
+DESCRIPTION="Tools for managing the osinfo database"
+HOMEPAGE="https://libosinfo.org/"
+SRC_URI="https://releases.pagure.org/libosinfo/${P}.tar.xz"
+
+LICENSE="GPL-2+"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~ia64 ~loong ~ppc ~ppc64 ~riscv ~sparc ~x86"
+IUSE="test"
+RESTRICT="!test? ( test )"
+
+RDEPEND="
+	>=dev-libs/glib-2.44:2
+	dev-libs/json-glib
+	>=app-arch/libarchive-3.0.0:=
+	net-libs/libsoup:3.0
+	>=dev-libs/libxml2-2.6.0
+"
+# perl dep is for pod2man (manpages)
+DEPEND="${RDEPEND}"
+BDEPEND="
+	virtual/pkgconfig
+	>=sys-devel/gettext-0.19.8
+	dev-lang/perl
+	test? (
+		$(python_gen_any_dep '
+			dev-python/pytest[${PYTHON_USEDEP}]
+			dev-python/requests[${PYTHON_USEDEP}]
+		')
+	)
+"
+
+PATCHES=(
+	"${FILESDIR}"/${PN}-1.10.0-no-clobber-fortify-source.patch
```

Comment:
Unfortunately, this is going to break with libxml2-2.12.0. Incidentally, I made the following pull request earlier today which also bumps osinfo-db-tools version:

https://github.com/gentoo/gentoo/pull/34061/files#diff-55c4047d343fd87b1ee1e97ce907a8138c08efd9f21a8cb25af08e79644cff07R43